### ISSUE TITLE
twisted/scripts/trial.py does not have a Python3 compatible syntax

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7154]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#7154|
|Type|enhancement|
|Created|2014-04-15 15:36:48Z|
|Branch|https://github.com/twisted/twisted/tree/trial.py-py3-7154|

This blocks the Python 3 migration.

<details><summary>Searchable metadata</summary>

```
trac-id__7154 7154
type__enhancement enhancement
reporter__Alex Alex
priority__normal normal
milestone__Python_3_x Python-3.x
branch__branches_trial_py_py3_7154 branches/trial.py-py3-7154
branch_author__radix radix
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1397576208000000 1397576208000000
changetime__1474198049187875 1474198049187875
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|Alex commented|
|-|-|

Patch is available at https://github.com/twisted/twisted/pull/11

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_2"></a>|@radix commented|
|-|-|

(In [42248]) Branching to trial.py-py3-7154.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_3"></a>|@radix commented|
|-|-|

Looks good, I'll merge it if the builds pass.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_4"></a>|@radix set status to `closed`|
|-|-|

(In [42253]) Merge trial.py-py3-7154: update trial.py for python3 syntax

Author: Alex
Reviewer: radix
Fixes: [#7154](https://github.com/twisted/twisted/issues/7154)

Update twisted/scripts/trial.py to have Python3-compatible syntax.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
